pypeit/core/gui/skysub_regions.py

```diff
@@ -571,22 +571,41 @@ def operations(self, key, axisID):
         self.replot()
 
     def get_result(self):
-        """Save a mask containing the skysub regions, and print information
+        """Generate a master calibration file containing a mask of the skysub regions, and print information
```

Comment:
please remove `master`

Reply:
Done (and I've also removed some other instances of this, as well)!